src/app/content/highlights/components/utils/showConfirmation.tsx

```diff
@@ -2,13 +2,16 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import { RawIntlProvider } from 'react-intl';
 import uuid from 'uuid/v4';
-import { AppServices } from '../../../../types';
+import createIntl from '../../../../messages/createIntl';
 import { assertDocument, assertNotNull } from '../../../../utils';
 import ConfirmationModal from '../ConfirmationModal';
 
-export default async(services: AppServices) => {
+export default async(services: any) => {
```

Comment:
@TomWoodward This is a placeholder as I'm confused about how to deal with typing given the change to `useServices`. Should I update create a new type that extends AppServices to include the redux middleware?

Reply:
you could just type this here as `AppServices & MiddlewareAPI`, pretty sure that is what the redux effect type does